meta-mender-commercial/recipes-mender/mender-monitor/mender-monitor.inc

```diff
@@ -18,11 +18,14 @@ FILES_${PN}_append_mender-systemd = " \
     ${systemd_system_unitdir}/mender-monitor.service \
 "
 
-S ?= "${WORKDIR}/${PN}"
+S = "${WORKDIR}/mender-monitor"
 
 do_version_check() {
     if ! ${@'true' if d.getVar('MENDER_DEVMODE') else 'false'}; then
-        bbfatal "Not yet released. This recipe can only be used with MENDER_DEVMODE"
+        tarball_version=$(cat ${S}/.version)
```

Comment:
do we want to check here if the `${S}/.version` file exists before calling cat?
```bash
tarball_version=""
[ -f "${S}/.version" ] && tarball_version=$(cat ${S}/.version)
```

Reply:
It doesn't really matter, it will fail in both cases...